src/bin/rustfmt.rs

```diff
@@ -9,17 +9,37 @@
 // except according to those terms.
 
 #![cfg(not(test))]
+#![feature(path_ext)]
 
 extern crate rustfmt;
 
 use rustfmt::{WriteMode, run};
 
-use std::fs::File;
-use std::io::Read;
+use std::env;
+use std::fs::{File, PathExt};
+use std::io::{self, Read, Error, ErrorKind};
+use std::path::PathBuf;
+
+fn lookup_config_file() -> io::Result<PathBuf> {
```

Comment:
Could you add a comment describing how the file is looked up please? Also, it might be better named `lookup_default_config_file` since, aiui, this only looks for default files up the tree, rather than project specific files, etc.